src/open_inwoner/templates/pages/contactmoment/list.html

```diff
@@ -39,4 +65,33 @@ <h1 class="utrecht-heading-1" id="contactmomenten">{{ page_title }} ({{ contactm
     {% if contactmomenten %}
         {% pagination page_obj=page_obj paginator=paginator request=request %}
     {% endif %}
+
+    <section class="contactmomenten contactmomenten__contact-form">
+        <div class="form-container">
+            <div class="form-heading">
+                <h2 class="h2">{% trans "Stel een vraag" %}</h2>
+                <p class="p">
+                {% blocktrans trimmed %}
+                    Heeft u een vraag? Dan kunt u deze hier stellen.
+                    Een van onze medewerkers beantwoord uw vraag z.s.m.
+                {% endblocktrans %}
+                </p>
+            </div>
+            <div>
+                <form action="{% url 'cases:contactmoment_list' %}" method="POST" id="question-form" class="contact-form">
+                    {% csrf_token %}
+
+                    {% for field in form.fields %}
```

Comment:
The "expand_more" icon is missing from the select-input here, so you may need to expand the form here into its separate fields. (Examples of the icon can be found in Samenwerken etc.)
 
<img width="807" alt="select-caret" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/fa398571-7426-4a58-b09e-2953be57f059">